### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Matthieu96Code**
- Add missing elements to the project
- Style matches Zeplin's design
- Use of CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -12,28 +13,36 @@ const Book = ({ data, id }) => {
...
+      <div className="book-progress">
+        <AiOutlineLoading3Quarters className="progress-bar" size="4rem" color="#0290ff" />
+        <div className="progress-text">
+          <p className="percent-complete">
+            {Math.floor(Math.random(100) * 100)}
+            %
+          </p>
+          <p className="complete">Completed</p>
+        </div>
       </div>
```
Comment (danmainah) [inline src/components/Book.js]: - [x] @Matthieu96Code great work on your design, however, the progress bar color doesn't match the design given:

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image  | image

File: src/components/Home.js [hunk 2]
```diff
@@ -51,7 +60,8 @@ const Home = () => {
   return (
     <>
       {bookList}
-      <h1>Book list empty</h1>
+      <div className="bar-horiz" />
+      <h1 className="book-state">Book list empty</h1>
       <AddBook />
     </>
```
Comment (danmainah) [inline src/components/Home.js:66]: - [ ] @Matthieu96Code kindly check on your form, it's going beyond the background, it should have a space left below the inputs, kindly check the comparison below for a better understanding

Zeplin Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Matthieu96Code , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by prolajumokeoni — APPROVED** [review]
Hi @Matthieu96Code 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
 - All changes requested have been implemented 🥇  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**